README.md

```diff
@@ -3,8 +3,6 @@ About qutip
 
 Home: http://qutip.org
 
-GitHub: https://github.com/qutip/qutip
```

Comment:
I'd like to get this back again, but it will require a change to conda-smithy. That might be a few weeks off, but in the mean time, you could always add it back again after merging this.